.circleci/config.yml

```diff
@@ -47,3 +47,48 @@ jobs:
 
       - store_artifacts:
           path: doc/build/html
+
+      - persist_to_workspace:
+          root: doc/build
+          paths:
+            - html
+
+  deploy:
+    working_directory: ~/repo
+    docker:
+      - image: circleci/python:3.8.3-buster
+
+    steps:
+      - attach_workspace:
+          at: /tmp/build
+
+      - add_ssh_keys:
+          fingerprints:
+            - "e5:04:e6:c4:d6:c3:34:b9:02:e6:9f:25:1c:01:9e:e9"
+
+      - run:
+          name: upload_devdocs
+          command: |
+            set -e
+            git clone git@github.com:networkx/documentation.git
+            cd documentation
+            rm -rf latest
+            cp -R /tmp/build/html latest
+            touch latest/.nojekyll
```

Comment:
This isn't needed.  There is already a ``.nojekyll`` in the directory containing ``latest``.